lib/lockmaster.js

```diff
@@ -109,10 +110,17 @@ Lockmaster.prototype.start = function start() {
         };
 
         // Start running at intervals.
-        me._running[river] = setInterval(riverRunner, intervalSeconds);
+        if (river.config.hasOwnProperty('cronInterval')) {
+            _.each(river.config.cronInterval, function(interval) {
+                schedule.scheduleJob(interval, riverRunner);
+            });
+        } else {
+            me._running[river] = setInterval(riverRunner, intervalSeconds);
+
+            // Run once now at startup.
+            riverRunner();
```

Comment:
Shouldn't this be outside the if/else statement? Otherwise only interval rivers are run on startup.